src/compiler/commandLineParser.ts

```diff
@@ -312,9 +312,19 @@ namespace ts {
             }
         },
         {
-            name: "traceModuleResolution",
+            name: "typesSearchPaths",

```

Comment:
at this point this is tsconfig only option and since it lacks description it is already excluded from the list of possible command line options. I think all these hides it well enough/